app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-Users lists
+<div class="users-body">
+<% @users.each do |user| %>
+  <a href='/users/<%= user.id %>' class='user-link'>
+    <div class=user-image></div>
```

Comment:
> User#index OK ✔️ 

![image](https://github.com/V0pe/BlogApp/assets/38088818/f1b9e2de-c724-4528-a82f-9e631f643005)

[OPTIONAL] It would have been nice if you had fixed the User `images` during this milestone. But since you have already completed the whole Blog App before, I'm sure you fixed it later on...